cluster/cluster.yaml

```diff
@@ -277,6 +277,14 @@ Resources:
           FromPort: 9999
           IpProtocol: tcp
           ToPort: 9999
+        - CidrIp: "{{.Values.vpc_ipv4_cidr}}"
+          FromPort: 9990
+          IpProtocol: tcp
+          ToPort: 9990
+        - CidrIp: "{{.Values.vpc_ipv4_cidr}}"
+          FromPort: 9990
+          IpProtocol: udp
+          ToPort: 9990
```

Comment:
This should be conditionally enabled only when clusterratelimit feature is enabled.